### PR TITLE
Create ACQF.json

### PR DESCRIPTION
**Pull request by siuc-nate**
First pass at creating a JSON-LD example for the ACQF

### CODE REVIEW

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:5]: Is `???` expected?

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:176]: Is `???` expected?

Comment (jeannekitchens) [reply QualificationsFramework/Examples/ACQF.json:176]: Asume using the ceterms names space.  QFs are not part of ASN so it doesn't make sense to use the ASN name space.

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:12]: Is this in our google drive?

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:71]: `Refers`?

Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:71]: Remove extra space after `problems/`?

Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:71]: The second and third bullets don't sound like "sub-domains". Consider rephrasing them appropriately.

Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:71]: Did you mean `Types of skills`?

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:85]: `from lower to higher levels. They are supported by...`

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:120]: The word `Simple` repeats four times. Necessary? :)

File: QualificationsFramework/Examples/ACQF.json
Comment (rohit-joy) [inline QualificationsFramework/Examples/ACQF.json:99]: `include: `: Is there more to be added?

**Comment by rohit-joy** [comment]
@siuc-nate Please have a look at the suggestions. I didn't mark all of them, so please review all the text.

**Comment by philbarker** [comment]
On the ??? comments:  these are useful in drafts like this as place holders for namespace abbreviations that haven't been decided yet.  

None of the comments on the text values are relevant. The text should reflect what is in the QF being represented, it's not ours to edit.

On the whole, I think that code review in this repository is not necessary. We use this for discussions and sharing ideas. As far as I know, nothing is deployed from it. Code review creates unnecessary work and slows that down.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:71]: I would keep the fill text of the definitions in the source, which all start with "In the context of ACQF..."

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:85]: Not ours to fix.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:71]: Not ours to fix.

Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:71]: Not ours to fix.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:99]: We need to discuss what to do in cases like this.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:120]: Not ours to fix.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [inline QualificationsFramework/Examples/ACQF.json:10]: Cover page shows "Version 17 November 2022"---remember how the rest of the world does dates :-) 
p2, version control shows this as version 6 .  

I would record the former as date valid from, the latter as the versionCode.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [inline QualificationsFramework/Examples/ACQF.json:20]: Add authors/contributors & publisher?

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [inline QualificationsFramework/Examples/ACQF.json:30]: for info: as well as the abbreviations used at the front of the doc, there is a Glossary in Annex 1. It is the latter that I had in mind for the Glossary concept scheme. This is valid too. Does anyone think we need to make a distinction between "abbreviations used" and "glossary" in our vocabulary?

**Comment by siuc-nate** [comment]
Based on the comments above and in emails, should we keep the ??? or use ceasn: for them for now? @jeannekitchens @philbarker 

I'll go over the other stuff here as well, thanks.

**Comment by philbarker** [comment]
Jeanne's suggestion was `ceterms:` which is the way I was leaning too.  I think it'll be safe to use that.

File: QualificationsFramework/Examples/ACQF.json
Comment (siuc-nate) [reply QualificationsFramework/Examples/ACQF.json:12]: It should be fine for now.  It's just an internal document for discussion.

File: QualificationsFramework/Examples/ACQF.json
Comment (siuc-nate) [reply QualificationsFramework/Examples/ACQF.json:71]: I had copied these definitions from the paragraphs on page 21, which I now notice is slightly different from the text in the table on page 19.  I can use the table text instead here if that's preferable.

Comment (siuc-nate) [reply QualificationsFramework/Examples/ACQF.json:71]: I'm not opposed to it, but the space is there in the PDF (mid-line, so it's not an artifact of a line break).  We should discuss the impact of fixing (or not) these kinds of minor typographical things with @philbarker since part of what we want to do is capture the data as-is.

File: QualificationsFramework/Examples/ACQF.json
Comment (siuc-nate) [reply QualificationsFramework/Examples/ACQF.json:20]: Would we add everyone under "Authors and reviewers of this policy document" and/or "authors and reviewers of the ACQF guidelines..." or just one of those sets?  And, would we include contributors under "authors"?

I'm not quite sure what organization to attribute publishing to.

**Comment by siuc-nate** [comment]
I'll use ceterms for now, but I think ceasn makes more sense. Thanks.

File: QualificationsFramework/Examples/ACQF.json
Comment (siuc-nate) [reply QualificationsFramework/Examples/ACQF.json:44]: I added another scheme for that glossary and updated the JSON for the abbreviations.

I don't think a distinction is really necessary, and I'm sure there would be cases where abbreviations and definitions get intermingled in the same scheme.

File: QualificationsFramework/Examples/ACQF.json
Comment (philbarker) [reply QualificationsFramework/Examples/ACQF.json:34]: I wouldn't sweat it too much at this stage, we just need to show that we can list those things. Presumably anyone publishing their own QF as data would know who wrote and published it.

**Comment by jeannekitchens** [comment]
I have to say here that using the PR approach here is useful to have a way to flag and resolve specific questions etc.. and code changes.  I think it's very useful to have the comments in one place and see the resolutions so I would not drop this process.

**Comment by philbarker** [comment]
@siuc-nate @jeannekitchens the doc we have is a Policy Document that quotes the QF.  That makes it difficult to get some of the properties right: is the publication date of the Policy Doc the same as the QF? Are the authors the same?  But such documents are useful, so do we need a way of referencing them from the QF?

**Comment by jeannekitchens** [comment]
@philbarker @siuc-nate we have to ask Eduarda specifically about the ACQF. She is the expert who led the development of it.   For this example, we should go with what we see on the face of the policy doc.  However, the document is via the ACQF website https://acqf.africa/.  Per the website, the owner of the policy and all related guidelines is the African Union https://au.int//.  The document shows the authors and contributors.  The owner and publisher, I'd identify as the AU and we can have all of these questions in a list for Eduarda.

**Comment by siuc-nate** [comment]
@jeannekitchens Do we need to do that before we can merge this PR?

**Comment by siuc-nate** [comment]
Thanks @jeannekitchens 

I updated the JSON to reference the AU as an organization that is the creator/publisher of the QF.

I'll need someone else to approve the PR in order to merge it, unless there's something else that needs changing.

**Comment by jeannekitchens** [comment]
Have Rohit or Phil approve it.

**Comment by siuc-nate** [comment]
@philbarker @rohit-joy Please see @jeannekitchens's message above

**Review by philbarker — APPROVED** [review]

**Pull request merged**
